spec/classes/redis_spec.rb

```diff
@@ -81,6 +81,21 @@ class { 'redis::globals':
             end
           end
         end
+
+        describe 'with manage_dnf_module true', if: facts[:os]['family'] == 'RedHat' && facts[:os]['release']['major'].to_i > 7 do
+          let(:pre_condition) do
+            <<-PUPPET
+            class { 'redis::globals':
+              manage_dnf_module => true,
+              version           => '7',
+            }
+            PUPPET
+          end
+
+          it { is_expected.to compile.with_all_deps }
+
```

Comment:
Could you add the expectation `contain_package('redis dnf module').with_ensure('7')`